practice-one/src/components/CustomerItem/CustomerItem.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from "react";
+import "./CustomerItem.css";
+import { menuDot } from "@assets/images";
+import { CustomerInfo, Gender, ContextMenu } from "@components";
+
+const CustomerItem = ({ avatar, name, email, phoneNumber, gender }) => {
+  const [isShow, setIsShow] = useState(false);
```

Comment:
what will show?
Please create state or variable meaningful variables

Reply:
I changed the name 
![image](https://github.com/datdragon777/React-Training/assets/74806826/48259927-5d90-413d-9399-c34d58f940e1)